File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = 0;
    do
    {
        height = get_int("height : ");
    }
    while (height < 0);

    int blocks = 1;
    for(int i = 0;i < height;i++)
    {

        int spaces = height-blocks;
        for(j = 0;j<spaces;j++)
        {
            printf(" ");
        }
        printf("\n");
    }
}

File: 6_Python/pset/mario/more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height in range(1, 9):
        break

for i in range(1,height+1):
    print(" "*(height-i), end="")
    print("#"*(i), end="")
    print("  ")
    print("#"*(i), end="")
    print()
